elyra/util/operation_parser.py

```diff
@@ -111,53 +99,89 @@ def __init__(self, operation_filepath):
     def type(self):
         raise NotImplementedError()
 
+    @property
+    def model(self):
+        return self.build_model()
+
     @property
     def operation_filepath(self):
         # TODO : Ensure this is getting the abs path
         return self._operation_filepath
 
+    @property
+    @abstractmethod
+    def regex_dict(self) -> Dict[str, List]:
+        # a dictionary of matching regex expressions to use keyed to the json stanza
+        # Parsing for ENVs e.g. regex_dict['NAME_OF_JSON_STANZA'] = ['REGEX_EXP_1', 'REGEX_EXP_2']
+        raise NotImplementedError()
+
     @abstractmethod
-    def env_list(self):
+    def file_as_code_chunks(self) -> List[str]:
```

Comment:
Seems like this should be more of a 'get next chunk' thing, than a list of chunks based on the way `build_model` is written.  A chunk could still be a list of strings, whether the list be a single item or not is up to the subclass.